covid_graphs/covid_graphs/prediction_generator.py

```diff
@@ -44,31 +44,16 @@ def generate_predictions(filename: Path, output_dir: Path) -> None:
     country_report = create_report(filename)
     last_date_dates = country_report.dates[-PREDICTION_DAYS:]
     fitted_formulas = {
-        last_data_date: FittedFormula(until_date=last_data_date, country_report=country_report)
+        last_data_date: FittedFormula(last_data_date=last_data_date, country_report=country_report)
         for last_data_date in last_date_dates
     }
 
     short_country_name = country_report.short_name
-    arg_parameters = CountryAtgParameters()
-    arg_parameters.long_country_name = country_report.long_name
-    arg_parameters.short_country_name = short_country_name
+    country_atg_parameters = CountryAtgParameters()
+    country_atg_parameters.long_country_name = country_report.long_name
+    country_atg_parameters.short_country_name = short_country_name
     for last_data_date, formula in fitted_formulas.items():
-        # TODO(lukas): create a serialization function inside the trace generator
```

Comment:
last_data_data is not used in this block

Reply:
Fixed. I've made fitted_formulas into a list.